scootapi/client/run_job_cmd.go

```diff
@@ -29,6 +29,10 @@ func (c *runJobCmd) registerFlags() *cobra.Command {
 func (c *runJobCmd) run(cl *Client, cmd *cobra.Command, args []string) error {
 	log.Println("Running on scoot", args)
 
+	if args == nil || len(args) == 0 {
+		return errors.New("a job id must be provided")
```

Comment:
I think this is no longer true with Ryan's changes (which you've incorporated below)